docker/localsettings.py

```diff
@@ -100,12 +100,11 @@
     'host': redis_host,
 }
 
-ELASTICSEARCH_HOST = 'elasticsearch2'
-ELASTICSEARCH_PORT = 9200  # ES 2 port
-ELASTICSEARCH_MAJOR_VERSION = 2
+ELASTICSEARCH_HOST = 'elasticsearch5'
+ELASTICSEARCH_PORT = 9200  # ES 5 port
+ELASTICSEARCH_MAJOR_VERSION = 5
 
-if os.environ.get('ELASTICSEARCH_MAJOR_VERSION'):
-    ELASTICSEARCH_MAJOR_VERSION = int(os.environ.get('ELASTICSEARCH_MAJOR_VERSION'))
+ELASTICSEARCH_MAJOR_VERSION = 5
```

Comment:
Can this duplicate line be removed?

Reply:
[945dfd8](https://github.com/dimagi/commcare-hq/pull/33485/commits/945dfd8a6dd1adcc4219e91318563c2dad305478)